ONSdigital/census-rm-case-processor
# added cols field_cord_id, field_officer-id and ce-capacity

# Motivation and Context
3 new fields needed for final rehearsal spec  & changed rgn to region for internal consistency

# What has changed
Added 3 new fields to expected input, and to output.
 private String fieldOfficerId;
  private String ceExpectedCapacity;
  private String collectionExerciseId;

These are now expected on the sample create msg, to be persisted and to be emitted

# How to test?
Using other branches from this PR, 
mvn clean install to check unit tests / ITs
then with other PR images bring up docker-dev and run the new acceptance tests

# Links
https://trello.com/c/CzpZXxpZ/741-update-case-data-model-to-reflect-final-rehearsal-sample-8
CASE API:
https://github.com/ONSdigital/census-rm-case-api/tree/rehearsal-sample-spec
ACTION SCH:
https://github.com/ONSdigital/census-rm-action-scheduler/pull/6/files
CASE-PROCERSSOR:
https://github.com/ONSdigital/census-rm-case-processor/pull/11
& new tests:
https://github.com/ONSdigital/census-rm-acceptance-tests/tree/rehearsal-sample-spec
